net.kvirc.KVIrc.yaml

```diff
@@ -8,7 +8,7 @@ command: kvirc
 
 finish-args:
   - --share=network
-  - --socket=x11
+  - --socket=wayland
```

Comment:
You need fallback-x11 too otherwise the app breaks on Xorg sessions

Reply:
Done